perllib/FixMyStreet/Cobrand/Gloucestershire.pm

```diff
@@ -102,7 +102,7 @@ sub contact_extra_fields { [ 'display_name' ] }
 =cut
 
 sub new_report_detail_field_label {
-    'Where is the location of the problem, and can you give us a little more information?'
+    'Where is the location of the problem? Please be specific and identify the nearest property address or landmark to the problem.'
```

Comment:
Does providing an address potentially conflict with our house rules?

Reply:
I think it's not a problem if it's just for location. @dracos didn't seem to have an issue with it in the ticket